CONTRIBUTING.md

```diff
@@ -130,11 +133,30 @@ func (ssa *SQLStorageAuthority) AddPerson(p Person) (error) {
 }
 ```
 
-Add a migration with
+You will also need to update the `initTables` function from `sa/database.go` to
+tell Gorp which table to use for your versioned model structs. Make sure to
+consult the flag you defined so that only **one** of the table maps is added at
+any given time, otherwise Gorp will error.  Depending on your table you may also
```

Comment:
With both `certStatusModelv1` and `certStatusModelv2` mapped to the `certificateStatus` table at the same time calling `dbMap.Insert(certStatusModelv1{})` produced:

```
panic: No ColumnMap in table certificateStatus type certStatusModelv2 with field Serial`
```